File: Modul5/C/PRAK504-2310817210020-MuhammadHasbiyanRusyadi.c
#include <stdio.h>
int reverse(int angka){
    int rvrsd = 0;
    while(angka != 0) {
        rvrsd = (rvrsd) * 10 + (angka % 10) ;
        angka /= 10;}
    return rvrsd;}
int main() {
    int A, B;
    scanf("%d %d",&A,&B);
    A = reverse(A);
    B = reverse(B);
    int C = A+B;
    printf("%d",reverse(C));}

File: Modul5/Python/PRAK504-2310817210020-MuhammadHasbiyanRusyadi.py
def reverse(angka) :
    rvrsd = 0
    while(angka != 0) :
        rvrsd = rvrsd * 10 + (angka % 10)
        angka //= 10
    return rvrsd
A,B = map(int,input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))



